databuilder/databuilder/extractor/dashboard/mode_analytics/mode_dashboard_extractor.py

```diff
@@ -20,6 +20,8 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DASHBOARD_GROUP_ID_TO_SKIP = 'dashboard_group_id_to_skip'
```

Comment:
if this is a list can it be plural ? `DASHBOARD_GROUP_IDS_TO_SKIP` ?

Reply:
Sure. Will change